scripts/dependencies_check

```diff
@@ -6,25 +6,29 @@
 # named for the tool (i.e., qemu-user-static).
 dependencies_check()
 {
-	local depfile deps missing
+	local depfile deps missing missing2
 
 	for depfile in "$@"; do
 		if [[ -e "$depfile" ]]; then
 			deps="$(sed -f "${SCRIPT_DIR}/remove-comments.sed" < "${BASE_DIR}/depends")"
 
 		fi
 		for dep in $deps; do
-			if ! hash "${dep%:*}" 2>/dev/null; then
-				missing="${missing:+$missing }${dep#*:}"
+			if ! hash "$(echo ${dep} | sed 's/:.*//')" 2>/dev/null; then	
+				missing="$missing $(echo ${dep} | sed 's/^.*\(:.*:\).*$/\1/' | sed 's/://g')"
+				missing2="$missing2 $(echo ${dep} | sed 's/.*://')"
```

Comment:
Maybe a better name for these vars would be `missing_debian` and `missing_ubuntu`?

Reply:
I'll do that.